components/about.js

```diff
@@ -29,11 +31,57 @@ const Logo = styled.img`
 
 const CallToAction = LinkButton.extend`margin-top: 48px;`
```

Comment:
this should probably just be our `Button` component rather than `LinkButton` since it's not a link.